domain-models-runtime/src/test/java/org/folio/rest/tools/utils/EnvsTest.java

```diff
@@ -27,6 +27,8 @@ public void setUp() {
     map.put("DB_QUERYTIMEOUT", "8");
     map.put("DB_MAXPOOLSIZE", "5");
     map.put("DB_CONNECTIONRELEASEDELAY", "12345");
+    map.put("DB_RECONNECTATTEMPTS", "3");
+    map.put("DB_RECONNECTINTERVAL", "2000");
     map.put("DB_EXPLAIN_QUERY_THRESHOLD", "100");
     // we dropped support for dot form. check that it is ignored
```

Comment:
Minor. Maybe revise comment to `We dropped support for dot form. check that it is ignored.`

Reply:
Out of scope and not applicable for reconnection.

Create a separate pull request if you want this been changed.